src/main/java/com/exasol/adapter/dynamodb/queryrunner/PlanDoesNotFitException.java

```diff
@@ -0,0 +1,12 @@
+package com.exasol.adapter.dynamodb.queryrunner;
+
+/**
+ * This Exception is thrown by a QueryPlanFactory if the plan does not fit for the given query.
```

Comment:
```suggestion
 * This exception is thrown by a {@link DynamodbGetItemQueryPlanFactory} if the plan does not fit for the given query.
```

Reply:
I used QueryPlanFactory here intentionally as in the future there will be different QueryPlanFactories that throw this exception.